hyp3_sdk/hyp3.py

```diff
@@ -64,7 +61,7 @@ def find_jobs(self, start: Optional[datetime] = None, end: Optional[datetime] =
             if param_value is not None:
                 if isinstance(param_value, datetime):
                     if param_value.tzinfo is None:
-                        param_value.replace(tzinfo=timezone.utc)
+                        param_value = param_value.replace(tzinfo=timezone.utc)
```

Comment:
should add a test for this fix around https://github.com/ASFHyP3/hyp3-sdk/blob/develop/tests/test_hyp3.py#L71

load 2-3 jobs into the database with different `request_time` values, then do a couple `find_jobs(start=..., end=...)` calls with and without timezones